base/linalg/generic.jl

```diff
@@ -1268,27 +1318,34 @@ end
 end
 
 """
-    normalize(v, [p::Real=2])
+    normalize(v::AbstractVector, p::Real=2)
 
-Normalize the vector `v` with respect to the `p`-norm.
+Normalize the vector `v` so that its `p`-norm is equal to unity,
```

Comment:
Though I would be happy either way, the [documentation on documentation](https://docs.julialang.org/en/stable/manual/documentation/) (specifically the third numbered convention) advises against repeating information shown in the method signature (and particularly type information) in the method description proper. Best!